pkg/acsengine/testdata/vnet/kubernetesvnet.json

```diff
@@ -2,7 +2,12 @@
   "apiVersion": "vlabs",
   "properties": {
     "orchestratorProfile": {
-      "orchestratorType": "Kubernetes"
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "dnsServiceIp": "172.17.1.10",
```

Comment:
To make the naming convention consistent can you update this to: dnsService**IP** instead of dnsService**Ip** like in  firstConsecutiveStaticIP